records_mover/db/vertica/unloader.py

```diff
@@ -62,8 +62,7 @@ def unload(self,
             msg = str(e)
             if msg:
                 logger.warning(msg)
-            logger.warning(f"Could not export to S3 directly, falling back to SELECT")
-            return super().unload(schema, table, unload_plan, directory)
+            raise NotImplementedError('S3 currently required for Vertica bulk unload')
```

Comment:
A while back, the superclass would do unloading via SELECT.  Unloading via SELECT is now handled in the move algorithm via some code in the tables target, and the superclass just raised a NotImplementedException.

Given that, there's no point in delegating to the superclass, and we can make this an abstract method.